ModularTegustation/tegu_items/lc13unique_items.dm

```diff
@@ -783,3 +783,173 @@
+	icon_state = "clerkbot2"
+	icon_living = "clerkbot2"
+	faction = list("neutral")
+	health = 150
+	maxHealth = 150
+	melee_damage_type = RED_DAMAGE
+	armortype = RED_DAMAGE
+	damage_coeff = list(RED_DAMAGE = 0.9, WHITE_DAMAGE = 0.9, BLACK_DAMAGE = 0.9, PALE_DAMAGE = 1.5)
+	melee_damage_lower = 12
+	melee_damage_upper = 14
+	robust_searching = TRUE
+	stat_attack = HARD_CRIT
+	del_on_death = TRUE
+	attack_verb_continuous = "buzzes"
+	attack_verb_simple = "buzz"
+	attack_sound = 'sound/weapons/bite.ogg'
+
+/mob/living/simple_animal/hostile/clerkbot/Initialize()
+	..()
+	QDEL_IN(src, (120 SECONDS))
+
+/obj/item/powered_gadget/handheld_taser
+	name = "Handheld Taser"
+	desc = "A portable electricution device. Two settings, stun and slow. Automatically slows abnormalities instead of stunning them."
+	icon = 'ModularTegustation/Teguicons/teguitems.dmi'
+	icon_state = "gadget1"
+	default_icon = "gadget1"
+	batterycost = 1500
+	var/batterycost_stun = 3000
+	var/current_state = FALSE
+	var/hit_message= null
+
+/obj/item/powered_gadget/handheld_taser/attack_self(mob/user)
+	if((cell && cell.charge >= batterycost)&&(current_state))
+		current_state=FALSE
+		to_chat(user, "<span class='nicegreen'>The Gadget's light burns orange before clicking ready to Stun.</span>")
+	else if((cell && cell.charge >= batterycost_stun)&&(!current_state))
+		current_state=TRUE
+		to_chat(user, "<span class='nicegreen'>The Gadget's light blinks yellow before clicking ready to Slow.</span>")
+	else
+		to_chat(user, "<span class='notice'>The Gadget buzzes. Battery charge too low.</span>")
+	..()
+
+/obj/item/powered_gadget/handheld_taser/attack(var/mob/living/T, mob/living/user)
+	if(user.a_intent == INTENT_HARM)
+		if(isabnormalitymob(T))
+			if (cell && cell.charge >= batterycost)
+				hit_message= "<span class='userdanger'>[user] smashes the taser into [T].</span>"
+				cell.charge = cell.charge - batterycost
+				visible_message(hit_message)
+				T.apply_status_effect(/datum/status_effect/qliphothoverload)
+			else
+				to_chat(user, "<span class='notice'>The Gadget buzzes. Battery charge too low.</span>")
+		else
+			if ((cell && cell.charge >= batterycost_stun)&&(!current_state))
+				hit_message= "<span class='userdanger'>[user] smashes the taser into [T].</span>"
+				cell.charge = cell.charge - batterycost_stun
+				for(var/mob/living/carbon/human/person in view(7, user))
+					to_chat(person, hit_message)
+				T.apply_status_effect(/datum/status_effect/qliphothoverload)
+				T.Knockdown(1)
+			else if((cell && cell.charge >= batterycost)&&(current_state))
+				hit_message= "<span class='userdanger'>[user] smashes the taser into [T].</span>"
+				cell.charge = cell.charge - batterycost
+				visible_message(hit_message)
+				T.Stun(10)
+			else
+				to_chat(user, "<span class='notice'>The Gadget buzzes. Battery charge too low.</span>")
+	else
+		hit_message= "<span class='notice'>[user] lightly pokes [T] with the taser.</span>"
+		visible_message(hit_message)
```

Comment:
```suggestion
/obj/item/powered_gadget/handheld_taser
	name = "Handheld Taser"
	desc = "A portable electricution device. Two settings, stun and slow. Automatically slows abnormalities instead of stunning them."
	icon = 'ModularTegustation/Teguicons/teguitems.dmi'
	icon_state = "gadget1"
	default_icon = "gadget1"
	batterycost = 1500
	var/batterycost_stun = 3000
	var/batterycost_slow = 1500
	var/hit_message= null

/obj/item/powered_gadget/handheld_taser/attack_self(mob/user)
	if(!cell || cell.charge < batterycost_slow)
		to_chat(user, "<span class='notice'>The Gadget buzzes. Battery charge too low.</span>")
		return
	if(batterycost == batterycost_slow)
		if(cell.charge < batterycost_stun)
			to_chat(user, "<span class='notice'>The Gadget buzzes. Battery charge too low.</span>")
			return
		batterycost = batterycost_stun
		to_chat(user, "<span class='nicegreen'>The Gadget's light burns orange before clicking ready to Stun.</span>")
		return
	batterycost = batterycost_slow
	to_chat(user, "<span class='nicegreen'>The Gadget's light blinks yellow before clicking ready to Slow.</span>")

/obj/item/powered_gadget/handheld_taser/attack(var/mob/living/T, mob/living/user)
	if(user.a_intent != INTENT_HARM)
		hit_message = "<span class='notice'>[user] lightly pokes [T] with the taser.</span>"
		visible_message(hit_message)
		return
	hit_message = "<span class='userdanger'>[user] smashes the taser into [T].</span>"
	if(isabnormalitymob(T) && cell.charge >= batterycost_slow)
		cell.charge = cell.charge - batterycost_slow
		visible_message(hit_message)
		T.apply_status_effect(/datum/status_effect/qliphothoverload)
		return
	if (!cell || !cell.charge >= batterycost || isisabnormalitymob(T))
		to_chat(user, "<span class='notice'>The Gadget buzzes. Battery charge too low.</span>")
		return
	if (batterycost == batterycost_stun)
		T.apply_status_effect(/datum/status_effect/qliphothoverload)
		T.Knockdown(1)
	else
		T.Stun(10)
	cell.charge = cell.charge - batterycost
	visible_message(hit_message)
```